gitlab/v4/objects/groups.py

```diff
@@ -194,6 +194,21 @@ def delete_ldap_group_link(
         path += f"/{cn}"
         self.manager.gitlab.http_delete(path, **kwargs)
 
+    @cli.register_custom_action("Group")
+    @exc.on_http_error(exc.GitlabGetError)
+    def list_ldap_group_links(self, **kwargs):
+        """Add an LDAP group link.
+
+        Args:
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabGetError: If the server cannot perform the request
+        """
+        path = "/groups/%s/ldap_group_links" % self.get_id()
+        return self.manager.gitlab.http_get(path, **kwargs)
```

Comment:
As a note (for other reviewers), this is probably correct. I initially thought should probably use `http_list()`, but upstream docs have zero information on what is returned ☹️ With no docs I guess `http_get()` is probably correct.

@rayisbadat would be interesting if you could provide the return value from the API call.

Reply:
If you curl the API directly It returns an array of dicts as show below (I redacted out our ldap info)
```
[{"cn":null,"group_access":40,"provider":"ldapmain","filter":"(memberOf=cn=XXXXX,ou=groups,ou=XXXXX,dc=XXXXX,dc=XXXXX,dc=XXXXX)"}]                              `
```

The output doesnt seem to change if i call http_get or http_list in the code.  Both look like
```
[{'cn': None,
  'filter': '(memberOf=cn=XXXXX,ou=groups,ou=XXXXX,dc=XXXXX,dc=XXXXX,dc=XXXXX)',
  'group_access': 40,
  'provider': 'ldapmain'}]
<class 'list'>
```

Reply:
@nejch  What do you think, `http_get` or `http_list`. After seeing the data returned I'm leaning towards `http_list`, but not sure.

Reply:
Looking at https://github.com/python-gitlab/python-gitlab/pull/2367 which adds similar functionality, I'd even go for a full manager with a `ListMixin`, since the number of custom methods here is growing. Both links endpoints seem like 2 very similar things. But maybe outside the scope of this PR - I'd go for `http_list` just in case though, for pagination.